recipe/meta.yaml

```diff
@@ -19,16 +19,18 @@ requirements:
         - hdf5
         - libnetcdf
         - udunits2
-        - krb5
         - expat
+        - krb5
+        - esmf
```

Comment:
We really need a nicer way of handling optional dependencies (nco still works without esmf even after this change, right?)